.metadocrc

```diff
@@ -0,0 +1,15 @@
+{
+  "files": [
+    "lib/core.js",
+    "lib/application.js"
+  ],
+
+  "customLinks": {
+    "IncomingMessage": "https://nodejs.org/dist/latest-v10.x/docs/api/http.html#http_class_http_incomingmessage",
```

Comment:
```suggestion
    "http.IncomingMessage": "https://nodejs.org/docs/api/http.html#http_class_http_incomingmessage",
```

Reply:
https://nodejs.org/docs/api/http.html#http_class_http_incomingmessage
```
404: Page could not be found
```

Did you mean https://nodejs.org/docs/latest/api/http.html#http_class_http_incomingmessage ?

Reply:
No, I meant https://nodejs.org/api/http.html#http_class_http_incomingmessage, but thanks for noticing.